run/calling_dash.py

```diff
@@ -133,17 +133,15 @@ def main():
         return
     last_updated = all_predictions_df["timestamp"].max()
     all_predictions_df = all_predictions_df.drop(columns="timestamp")
-    all_predictions_df.loc[
-        all_predictions_df["call_status"] == "Gebeld", "call_status"
-    ] = "🟢"
-    if "Wordt gebeld" in all_predictions_df["call_status"].values:
-        all_predictions_df.loc[
-            ~(all_predictions_df["call_status"] == "🟢"), "call_status"
-        ] = "📞"
-    all_predictions_df.loc[
-        ~all_predictions_df["call_status"].isin(["🟢", "📞"]),
-        "call_status",
-    ] = "🔴"
+    all_predictions_df["call_status"] = all_predictions_df["call_status"].fillna("🔴")
+    all_predictions_df["call_status"] = all_predictions_df["call_status"].replace(
```

Comment:
This doesn't work for all the different values, for example "Niet gebeld" is now just text on acc